service/src/main/java/bio/terra/tanagra/service/accesscontrol/impl/VumcAdminAccessControl.java

```diff
@@ -29,7 +29,7 @@ public String getDescription() {
   public boolean isAuthorized(
       UserId userId, Action action, ResourceType resourceType, @Nullable ResourceId resourceId) {
     return vumcAdminService.isAuthorized(
-        userId.getSubject(),
+        userId.getEmail(),
```

Comment:
nit: Can you add a comment to these two `VumcAdminService` methods describing what the method expects to be passed -- the subject id or the email?